### ISSUE TITLE
False-positive E1121: Too many positional arguments for function call (six.moves.reload_module)

### ISSUE THREAD

**Issue by pauleckhardt** [PyCQA/pylint#3509]
# Steps to reproduce

1. Create a file containing:

```
import sys
 import six
six.moves.reload_module(sys)
```

2. execute pylint filename

# Current behaviour

    E:  4, 0:    4,  0: E1121:    Too many positional arguments for function call (too-many-function-args)

# Expected behaviour

no E1121

# Temporary workaround

    """A"""
    import sys
    from six.moves import reload_module
    reload_module(sys)

# pylint --version

    pylint 2.4.4
    astroid 2.3.3
    Python 3.7.3 (default, Mar  6 2020, 22:34:30) 
    [Clang 11.0.3 (clang-1103.0.32.29)]
